uniffi_bindgen/src/bindings/python/templates/macros.py

```diff
@@ -5,19 +5,31 @@
 #}
 
 {%- macro to_ffi_call(func) -%}
-_UniFFILib.{{ func.ffi_func().name() }}({% call _arg_list_ffi_call(func.arguments()) -%})
+{%- match func.throws() -%}
+{%- when Some with (e) -%}
+_RustErrorHelper.try_raise(_RustErrorHelperPartial(_UniFFILib.{{ func.ffi_func().name() }},{% call _arg_list_ffi_call(func) -%}))
+{%- else -%}
+_UniFFILib.{{ func.ffi_func().name() }}({% call _arg_list_ffi_call(func) -%})
+{%- endmatch -%}
 {%- endmacro -%}
 
 {%- macro to_ffi_call_with_prefix(prefix, func) -%}
+{%- match func.throws() -%}
+{%- when Some with (e) -%}
+_RustErrorHelper.try_raise(_RustErrorHelperPartial(
+    _UniFFILib.{{ func.ffi_func().name() }},{{- prefix }}{% if func.arguments().len() > 0 %},{% endif %}{% call _arg_list_ffi_call(func) %})
+))
```

Comment:
Just a heads-up that you might run into indentation issues if you put newlines in your macro output like this, since you don't have any guarantee about what indentation level the invocation of the macro is at, and python is very strict about indentation.